pkg/security/podsecuritypolicy/selinux/mustrunas_test.go

```diff
@@ -54,13 +56,16 @@ func TestMustRunAsOptions(t *testing.T) {
 }
 
 func TestMustRunAsGenerate(t *testing.T) {
+	externalSELinuxOptions := &corev1.SELinuxOptions{
+		User:  "user",
+		Role:  "role",
+		Type:  "type",
+		Level: "level",
+	}
+	internalSELinuxOptions := &api.SELinuxOptions{}
+	v1.Convert_v1_SELinuxOptions_To_core_SELinuxOptions(externalSELinuxOptions, internalSELinuxOptions, nil)
```

Comment:
I didn't expect conversions here, just a change to working with external objects

Reply:
how about now? the conversion is replaced by struct instantiation